Gemfile.lock

```diff
@@ -108,10 +108,10 @@ GEM
     unicode-display_width (1.7.0)
 
 PLATFORMS
-  ruby
+  x86_64-darwin-19
```

Comment:
Is this change intentional? I am not familiar with the effects / consequences of changing the `PLATFORMS` section in the `Gemfile.lock`, so maybe this is ok, just checking. 😸